File: Day-1/program3.c
#include<stdio.h>
int main()
{//opening main
    int n;//initialising
    printf("enter number");
    scanf("%d",&n);//store input in variables
    for(int i=1;i<=n;i++)
    {//opening for loop
        if(n%i==0)
        printf("%d\n",i);
        else
        continue;
    }//closing for loop
    return 0;
}//closing main

File: Day-12/program3.java
import java.util.Scanner;
public class program3
{//opening class
    static void fibo(int n)//FUNCTION FOR FIBONACCI SERIES
     {
        int a=0,b=1,c;//initialising variables
        System.out.print("Fibonacci Series=");
        for (int i = 1; i <= n; i++) 
        {
            System.out.print(a + " ");
            c = a+b;
            a = b;
            b = c;
        }
    }
    public static void main(String[] args)
     {//opening main
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter the number of terms: ");
        int n = sc.nextInt();
        fibo(n); // Function call
        sc.close();
    }//closing main
}//closing class